migration/mule/service-orchestration-and-choice-routing/README.md

```diff
@@ -0,0 +1,143 @@
+
+#### Content-Based Routing 
+
+Using the Switch mediator, WSO2 provides the capability to route different messages into different flows of meditation. Switch mediator evaluates a given expression at run time and compares it against the given cases. The expression can be composed using the content of the message such as a certain attribute of the payload or header. If no case matches, the message will be routed to the default case.
+
+#### Service Orchestration 
+When it is required to make several web service calls to complete the processing of a single message, the process is commonly referred to as service orchestration. In this example, calls are made to get prices from different vendors and to provide order details to an auditing service.
+
+#### Asynchronous Message Processing
+In order to drive the latency to a minimum, it is better to execute parts of the process which are not necessary for the complete before the response is delivered. WSO2 provides the ability to asynchronously process parts of application via message stores. One can put messages to store during the mediation and assign a different process (sequence) to handle it. In this application, the auditing process demonstrates such a scenario which is an internal process executed for auditing services.
+
+### Assumption
+
+This document assumes that you are familiar with WSO2 EI, the 
+[Integration Studio interface](https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/), SOAP calls, and Web service development with WSDL. To 
+increase your familiarity with Integration Studio, consider completing one or more 
+[WSO2 EI Tutorials](https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/integration-use-cases/).
+
+### Example Use Case
+
+This application demonstrates the processing of orders in a store. Since an order can contain multiple items, that are processed iteratively. Price and other details of each item are obtained by making web service calls from respective vendors and using them final order details are calculated and a summary is sent back to the customer. Order details are also audited by a separate service and stored for future needs.
+
+### Set Up and Run the Example 
+
+As with the other example templates, you can create a fully-functioning applications straight out of the box in Anypoint Studio. You can tweak the configurations of these use case-based examples to create your own customized applications in Mule.
+The approach to running this example is using raw SOAP that bypasses the ordering process via the GUI. To achieve this, follow the procedure bellow:
```

Comment:
I am not quite getting what is meant by this statement. We can say, we will expose data about orders as a SOAP based web service and it using direct invocation.